conda/recipes/legate-dataframe/meta.yaml

```
@@ -96,11 +97,13 @@ outputs:
         # released, a new legate-dataframe would be required.
         - legate
         - numpy >=1.23,<3.0.0a0
+        - nccl {{ nccl_version }}
         - rmm {{ rapids_version }}
         - cudf {{ rapids_version }}
         - pynvjitlink<=0.6
     ignore_run_exports_from:
       - cupynumeric  # only a build dependency to help resolver
+      - nncl  # cargo-culted from raft (assume the pin is overly strict)
```

```suggestion
      - nccl  # cargo-culted from raft (assume the pin is overly strict)
```

This looks like a typo.

---

> _assume the pin is overly strict_

I'm not sure why `raft` recipes ignore the run exports on `nccl`. The `nccl` from `conda-forge` only has a major-version pin as its run export (https://github.com/conda-forge/nccl-feedstock/blob/d10b348347db1a1175654f10e6c24f5d524c8652/recipe/meta.yaml#L23) which shouldn't be too restrictive. Maybe that `ignore_run_exports` entry is to help in the case where we get `nccl` from the `nvidia` channel 🤷🏻 

I think it's fine if you want to ignore those run exports here too (as this PR is doing). But let's see what CI says once you fix this typo.

---

Should have checked, I'll remove the comment. Probably it's the opposite then: we want a newer NCCL, but the run-export doesn't ensure that.

---

Ok yeah, makes sense.